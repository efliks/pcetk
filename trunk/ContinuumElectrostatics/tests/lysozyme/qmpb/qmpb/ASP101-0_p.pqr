../0/site_ASP_101_p.pqr