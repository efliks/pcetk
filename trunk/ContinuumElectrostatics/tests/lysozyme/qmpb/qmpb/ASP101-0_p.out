Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named ASP101-0_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (-3.626, 24.588, 32.343) 101 1
Finer level:    (-3.626, 24.588, 32.343) 101 0.5
Finest level:   (-3.626, 24.588, 32.343) 101 0.25


Self energy of ASP101-0_p in three dielectric environment = 173.244
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of ASP101-0_p with background charge set in three dielectric environment = 1.18138
(probably in kcal/mole)



Interaction energies of ASP101-0_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.000394355
0   1 : 4.71075e-06
1   0 : -2.41534e-05
1   1 : 0.000284345
2   0 : -0.000776089
2   1 : -2.69079e-05
3   0 : -0.00325518
3   1 : -0.000116695
3   2 : -0.000222493
3   3 : 0.00263708
4   0 : -0.000288022
4   1 : 0.000808193
5   0 : -0.00529765
5   1 : 0.0308456
6   0 : -0.00017865
6   1 : -0.00725966
7   0 : 0.000179783
7   1 : 4.58659e-06
8   0 : -0.000187765
8   1 : -0.00507459
9   0 : -0.000370492
9   1 : -0.00540088
10   0 : -0.00075545
10   1 : -0.00626552
11   0 : -0.000148626
11   1 : 0.00115833
12   0 : -0.000314017
12   1 : 0.00125282
13   0 : -1.37796e-05
13   1 : 0.00169863
14   0 : -0.0119492
14   1 : -0.000141177
15   0 : -0.053872
15   1 : -0.00122879
16   0 : 346.488
16   1 : 452.876
17   0 : 0.00415318
17   1 : -2.92316e-05
18   0 : 6.92898e-06
18   1 : -0.000191046
Total runtime: 7.46 s user 0.06 s system 
